crates/hyperqueue/src/client/commands/mod.rs

```diff
@@ -6,3 +6,23 @@ pub mod server;
 pub mod submit;
 pub mod wait;
 pub mod worker;
+
+/// Helper macro for generating CLI help for a `Duration` (or `Option<Duration>`) value
+/// that can be specified either using the HMS or humantime formats.
+macro_rules! duration_doc {
+    ($text:expr) => {
+        concat!(
+            $text,
+            "\n\n",
+            r#"You can use either the `HH:MM:SS` format or a "humantime" format.
+For example:
+- 01:00:00 => 1 hour
+- 02:05:10 => 2 hours, 5 minutes, 10 seconds
+- 1h => 1 hour
+- 2h5m10s => 2 hours, 5 minutes, 10 seconds
+- "3h 10m 5s" => 3 hours, 10 minutes, 5 seconds (with spaces, wrapped in quotes)"#
```

Comment:
Quotes around time specification containing spaces depends on shell, isn't it?
I can also write "--heartbeat=3h 2s", but I cannot write '--heartbeat="3h 2s"'
Therefore claim that time specification with space has to be contain quotes is misleading.

Btw: You can also use full names of time units:
"2 hours 3 minutes"

Reply:
Ok, fair., removed the quotes from the example and removed the whole parentheses section. I hope that people will realize that they need to quote the time somehow when it contains spaces :)